src/utils/nonNull.ts

```diff
@@ -21,7 +22,7 @@ export function nonNullValue<T>(value: T | undefined, propertyNameOrMessage?: st
     if (isNullOrUndefined(value)) {
         throw new Error(
             // tslint:disable-next-line:prefer-template
-            'Internal error: Expected value to be neither null nor undefined'
+            localize('notNullUndefined', 'Internal error: Expected value to be neither null nor undefined')
```

Comment:
I don't want to localize internal errors - this is for our telemetry, not for showing the user.